### DISCUSSION TITLE
No Odometer in Charges

### DISCUSSION THREAD

**Discussion by elizaso** [teslamate-org/teslamate#4109] [General]
I am running 1.30.1 version and I don't see the odometer information in **Charges** neither in **Charge Details**. Is it something wrong in my installation? Is it possible to fix it? Thank you in advance. Great SW!!

**Comment by rc-16** [comment]
Was it there before? 

I look at the Timeline dashboard to easily access odometer information

**Reply by elizaso** [reply]
That's correct, but I would like to know the odometer value in each charge

**Comment by cwanja** [comment]
Does look like it was removed in #4031 @jheredianet. 

Can you share a bit more about your use-case for seeing odometer in either dashboard? They are small snapshot in time. Charges is last three months and Charge Details is just the specific timeframe from the selected charge.

**Reply by elizaso** [reply]
My use-case is to know with which Odometer value I made each charge. As far as I see now, I cannot get it checking my charges. And I have to figure out when and where I made the charge to get the Odometer value at that time. For me to have this is important, and I assume for other users too.

**Reply by jheredianet** [reply]
Hi, I will try to get it back on next release.

**Reply by Enriko82** [reply]
Hi Elizaso,
I requested the same, but different last year. DrMichael created an updated 
[charges.json file](https://github.com/teslamate-org/teslamate/pull/2658)

See his Pull request [here](https://github.com/teslamate-org/teslamate/pull/3459)
Additionally I now use the timeline as proposed and filter on Charges, which is doing the same for me.

Kind regards,
Enriko
